### ISSUE TITLE
apu glitch

### ISSUE THREAD

**Issue by Matt726-S** [mSparks43/747-400#553]
when starting the apu, you have to hold down the start for way too long. the apu in this aircraft will only start when you push the start and then wait until 81% n2. either when you’ve held it in the start position for that long or if you move the apu selector to start again after its at 81 percent n2, then the n1 will rise. 

observe here, he doesn’t have to hold it for more than 4 seconds

**Comment by crazytimtimtim** [comment]
Yeah, I think we never really sorted out the APU issues. @Juppia904 just kinda...disappeared.

**Comment by crazytimtimtim** [comment]
Yeah, I think we never really sorted out the APU issues. @Juppia902 just kinda...disappeared.

**Comment by crazytimtimtim** [comment]
Yeah, I think we never really sorted out the APU issues. @Juppie902 just kinda...disappeared.

**Comment by crazytimtimtim** [comment]
I guess I'll fix this stuff up.

**Comment by MCCVeen** [comment]
> Yeah, I think we never really sorted out the APU issues. @Juppie902 just kinda...disappeared. I guess I'll try and sort this stuff out.

Have you started working on this yet? Hope this gets sorted before V2 including the unnecessary APU and APU DOOR fault indications @mSparks43 (Oh btw nice RF textures Tim, thanks for those!)
![image](https://user-images.githubusercontent.com/75425109/121296891-ae303300-c8f1-11eb-8a69-410973e813e9.png)

**Comment by Juppie902** [comment]
If you have a fix for it, go ahead, due to work I was cut short on free time (was working from home around the time I did some fixes)
I hope eventually I will get back to helping

**Comment by Matt726-S** [comment]
its really annoying cuz u have to either hold or wait to switch the apu start. please fix
